charts/lfx-platform/templates/heimdall/middleware.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright The Linux Foundation and each contributor to LFX.
+# SPDX-License-Identifier: MIT
+---
+{{ if .Values.heimdall.enabled -}}
```

Comment:
should this also an `and` clause for `traefik.enabled` -- it would be an edge case, but the middleware CRD API may not exist if the traefik chart has never been added.